src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java

```diff
@@ -1,10 +1,17 @@
 package edu.kis.vh.nursery.list;
 
+/**
+ * Podstawowa implementacja linked list używająca int jako typu danych elementów
+ */
 public class IntLinkedList {
 
 	Node last;
 	int i;
```

Comment:
Unused (private) field could be removed.

Reply:
Should be fixed now